.git-bash-for-mac-zsh.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/zsh
+
+autoload -Uz compinit && compinit
+source ~/.git-prompt.sh
+
+autoload -U colors && colors
+DEFAULT="%{$reset_color%}"
+GREEN="%{$fg[green]%}"
+YELLOW="%{$fg[yellow]%}"
+CYAN="%{$fg[cyan]%}"
+
+export PS1=$GREEN"$USER@%m"$YELLOW" %~"$CYAN'$(__git_ps1)'$DEFAULT$'\n'"\$ "
```

Comment:
I always see `(__git_ps1)` in branch name in catalina 10.15.7

Reply:
If you just type `__git_ps1`, do you see the branch name? Do you have the file `~/.git-prompt.sh` placed in your home directory? If you see the `install.sh`, it should be fetched with `curl`, and it is required for this command to work.

Reply:
i have the same issue @Ondkloss

Reply:
Thanks for the info @cagnulein. Did you check the two things I mentioned in the previous post? Also I'm assuming you somehow used my fork, or otherwise used this pull request?

Reply:
@ondkloss i used your fork. __git_ps1 gives me the right branch name. and yes i have the .git-prompt file.

Reply:
@cagnulein ty. I'll take a second look a bit later!

Reply:
@Ondkloss now it's working! thanks man!

Reply:
Sorry for the delay. Glad it worked @cagnulein!
Sorry for not fixing your issue earlier @hl1028, but it appears to be resolved now.